cowamm/kpi/competitors/uni_swap_style/largest_uni_style_syncs_4304212.sql

```diff
@@ -0,0 +1,28 @@
+-- This query finds all the sync event which happens after a swap and reference the new reserves of the pool
+-- Parameters:
+-- {{blockchain}}: The blockchain to query
+
+-- Finds the pools with the largest tvl
+with pools as (
+    select
+        pool_address,
+        token0,
+        token1
+    from "query_4303563(blockchain='{{blockchain}}', number_of_pools = '{{number_of_pools}}')"
+)
+
+select
```

Comment:
Is this query useful as its own standalone query? It looks like it's only used in one place and there we do a bunch of the joins we also do here, so it seems like it might be easier to inline it there?